EpiAware/test/test_utilities.jl

```diff
@@ -82,6 +82,7 @@ end
 end
 
 @testitem "Testing growth_rate_to_reproductive_ratio function" begin
+    using EpiAware
```

Comment:
why is this needed here but not elsewhere?

Reply:
Its not.

Reply:
Removed